markdown_flow/tag_filter.py

```
@@ -0,0 +1,82 @@
+"""
+preserve_or_translate 标签过滤器
+
+LLM 有时不遵守 system prompt 中的指令，会在输出中保留这些标签。
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)